src/ParticleIO/ESHDFParticleParser.cpp

```diff
@@ -86,7 +86,7 @@ bool ESHDFIonsParser::put(xmlNodePtr cur)
   }
   delete [] species_names;
   ParticleSet::Tensor_t lat(ref_.Lattice.R);
-  ParticleSet::Buffer_t pbuffer;
+  PooledData<double> pbuffer;
```

Comment:
Why double and not a parameterized type?

Reply:
This is to pack send/recv data for the builder. Safe to use highest precision. I will change it to the full precision macro.